### ISSUE TITLE
Is there docs about "Package.appxmanifest" of the UWP?

### ISSUE THREAD

**Issue by moyanming** [MicrosoftDocs/visualstudio-docs#1299]
Hi,
Is there detail docs about "Package.appxmanifest" of the UWP? I want to add some existing folder and files to my project, but there are no those folder and files after I deployment the app.
Add folder and files to my project (but I can't find those folder and files after deployment):
![qq 20180727114907](https://user-images.githubusercontent.com/13234209/43300708-47eb1530-9193-11e8-90b2-10ca29a40079.jpg)

The "Package.appxmanifest" is:
<?xml version="1.0" encoding="utf-8"?>
<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities" IgnorableNamespaces="uap mp rescap">
  <Identity Name="f3c8ac69-1f26-4c6d-94b0-0ff1492438a4" Publisher="CN=mym" Version="1.0.0.0" />
  <mp:PhoneIdentity PhoneProductId="f3c8ac69-1f26-4c6d-94b0-0ff1492438a4" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
  <Properties>
    <DisplayName>App1</DisplayName>
    <PublisherDisplayName>mym</PublisherDisplayName>
    <Logo>Assets\StoreLogo.png</Logo>
  </Properties>
  <Dependencies>
    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.0.0" MaxVersionTested="10.0.0.0" />
  </Dependencies>
  <Resources>
    <Resource Language="x-generate" />
  </Resources>
  <Applications>
    <Application Id="App" Executable="$targetnametoken$.exe" EntryPoint="App1.App">
      <uap:VisualElements DisplayName="App1" Square150x150Logo="Assets\Square150x150Logo.png" Square44x44Logo="Assets\Square44x44Logo.png" Description="App1" BackgroundColor="transparent">
        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png">
        </uap:DefaultTile>
        <uap:SplashScreen Image="Assets\SplashScreen.png" />
      </uap:VisualElements>
    </Application>
  </Applications>
  <Capabilities>
    <Capability Name="internetClient" />
    <uap:Capability Name="musicLibrary" />
    <uap:Capability Name="picturesLibrary" />
    <uap:Capability Name="videosLibrary" />
    <rescap:Capability Name="broadFileSystemAccess" />
  </Capabilities>
</Package>

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5f424cc3-c8c4-1066-0422-3d928bdd9783
* Version Independent ID: 4f9bcddd-7c3d-afd7-8bf2-b1df7f31c292
* Content: [How to: Create a Package Manifest - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/deployment/how-to-create-a-package-manifest)
* Content Source: [docs/deployment/how-to-create-a-package-manifest.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/deployment/how-to-create-a-package-manifest.md)
* Product: **visual-studio-dev15**
* GitHub Login: @Mikejo5000
* Microsoft Alias: **mikejo**

**Comment by moyanming** [comment]
<?xml version="1.0" encoding="utf-8"?>
<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities" IgnorableNamespaces="uap mp rescap">
  <Identity Name="f3c8ac69-1f26-4c6d-94b0-0ff1492438a4" Publisher="CN=mym" Version="1.0.0.0" />
  <mp:PhoneIdentity PhoneProductId="f3c8ac69-1f26-4c6d-94b0-0ff1492438a4" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
  <Properties>
    <DisplayName>App1</DisplayName>
    <PublisherDisplayName>mym</PublisherDisplayName>
    <Logo>Assets\StoreLogo.png</Logo>
  </Properties>
  <Dependencies>
    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.0.0" MaxVersionTested="10.0.0.0" />
  </Dependencies>
  <Resources>
    <Resource Language="x-generate" />
  </Resources>
  <Applications>
    <Application Id="App" Executable="$targetnametoken$.exe" EntryPoint="App1.App">
      <uap:VisualElements DisplayName="App1" Square150x150Logo="Assets\Square150x150Logo.png" Square44x44Logo="Assets\Square44x44Logo.png" Description="App1" BackgroundColor="transparent">
        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png">
        </uap:DefaultTile>
        <uap:SplashScreen Image="Assets\SplashScreen.png" />
      </uap:VisualElements>
    </Application>
  </Applications>
  <Capabilities>
    <Capability Name="internetClient" />
    <uap:Capability Name="musicLibrary" />
    <uap:Capability Name="picturesLibrary" />
    <uap:Capability Name="videosLibrary" />
    <rescap:Capability Name="broadFileSystemAccess" />
  </Capabilities>
</Package>

**Comment by moyanming** [comment]
![qq 20180727115337](https://user-images.githubusercontent.com/13234209/43300789-b1319122-9193-11e8-9988-a8e49f13c988.jpg)

**Comment by Mikejo5000** [comment]
@moyanming Yes, I think you want these docs: https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/appx-package-manifest
